autoscaler/controllers/gateway/deployment.go

```diff
@@ -42,8 +42,8 @@ func syncDeployment(dests *odigosv1.DestinationList, gateway *odigosv1.Collector
 		return nil, errors.Join(err, errors.New("failed to get secrets hash"))
 	}
 
-	// Calculate the hash of the config data and the secrets version hash, this is used to make sure the gateway will restart when the config changes
-	configDataHash := common.Sha256Hash(fmt.Sprintf("%s-%s", configData, secretsVersionHash))
+	// Use the hash of the secrets  to make sure the gateway will restart when the secrets (mounted as environment variables) changes
+	configDataHash := common.Sha256Hash(secretsVersionHash)
```

Comment:
Can we remove the `configData` argument from this function?

Reply:
that's what I did, since we don't want a change in the configdata to trigger a new rollout, we don't need to hash the config data anymore. is that what you were asking?

Reply:
I was referring to the function signature of `syncDeployment`:

```Go
func syncDeployment(dests *odigosv1.DestinationList, gateway *odigosv1.CollectorsGroup, configData string,
	ctx context.Context, c client.Client, scheme *runtime.Scheme, imagePullSecrets []string, odigosVersion string) (*appsv1.Deployment, error)
```

Reply:
Ah, updated in https://github.com/odigos-io/odigos/pull/1903/commits/6d53ab0d054b12fd942e23174945c2b6c9abcefc